birdhouse/deployment/trigger-deploy-notebook

```diff
@@ -12,12 +12,6 @@
 #
 # Logs to /var/log/PAVICS/notebookdeploy.log, re-use existing logrotate.
 
-
-if [ -z "$DOCKER_JUPYTERHUB_USER_PERSISTENCE_VOLUME" ]; then
-    # dupe with pavics-compose.sh
-    DOCKER_JUPYTERHUB_USER_PERSISTENCE_VOLUME=jupyterhub_user_persistence
```

Comment:
Since this script do not read `env.local`, I had to set the default value here.  It's a trade-off to make it independent of `env.local` or rather the location of `env.local`.  From memory this script is dropped directly into /etc/cron.hourly as a standalone script, not as a cronjob calling script with ability to pass the path to `env.local` as arguments.